### PR TITLE
Add widget control positions to docs

### PR DESCRIPTION
**Pull request by mtthwmnc**
Regarding this new feature: https://github.com/punkave/apostrophe/pull/969

Not sure the best place to put it within the docs. Thoughts?

### CODE REVIEW

File: docs/tutorials/getting-started/custom-widgets.md [hunk 1]
```diff
@@ -189,6 +189,27 @@ And pass the option in our `apos.area` call on `home.html`:
...
+```
+The available options are:
+
+`bottomLeft`
+
+`bottomRight`
+
+`topRight`
+
+`topLeft` (this is the default position, but you may use it for granularity)
```
Comment (abea) [inline docs/tutorials/getting-started/custom-widgets.md]: Might just need `(default)`

File: docs/tutorials/getting-started/custom-widgets.md [hunk 2]
```diff
@@ -189,6 +189,27 @@ And pass the option in our `apos.area` call on `home.html`:
...
+Depending on the your site design, you may want to position your widget controls to your liking. You can do this with options as well.
+
+```markup
+{{
+  apos.singleton(data.page, 'footer', 'my-footer', { position: "bottomLeft" })
+}}
+```
+The available options are:
+
+`bottomLeft`
```
Comment (abea) [inline docs/tutorials/getting-started/custom-widgets.md]: These should probably be an unordered list or just part of the paragraph.

Comment (bobclewell) [reply docs/tutorials/getting-started/custom-widgets.md]: +1 for a bulleted/unordered list here.

**Comment by mtthwmnc** [comment]
@tom enhanced this feature with a `controls: {}` object and this documentation is no longer relevant.

**Pull request closed without merging**